### PR TITLE
LowercaseKeywordsFixer - handle CT_CLASS_CONSTANT

### PR DESCRIPTION
**Pull request by tgabi333**
Two changes plus tests:
`CT_CLASS_CONSTANT ` is recognized case insensitive mode
`CT_CLASS_CONSTANT ` was added to list of known keywords

Closes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2070

### CODE REVIEW

**Comment by SpacePossum** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
Maybe add one more tokenizer unit test too?

**Comment by keradus** [comment]
like ?

**Comment by Soullivaneuh** [comment]
> like ?

Maybe `::CLASS`? But I'm not sure about the benefit...

**Comment by tgabi333** [comment]
@Soullivaneuh There is two test already, one is `::class` and the other is `::cLaSS` in `ClassConstantTest`

As i see another testcase would only test the case sensitve flag of `Token::equalsAny`

**Comment by keradus** [comment]
you didn't change `equalsAny` so no need to change the tests for it

**Comment by GrahamCampbell** [comment]
Looks good. 👍

File: Symfony/CS/Tokenizer/Token.php [hunk 1]
```diff
@@ -270,7 +270,7 @@ public static function getKeywords()
                 'T_INTERFACE', 'T_ISSET', 'T_LIST', 'T_LOGICAL_AND', 'T_LOGICAL_OR', 'T_LOGICAL_XOR',
                 'T_NAMESPACE', 'T_NEW', 'T_PRINT', 'T_PRIVATE', 'T_PROTECTED', 'T_PUBLIC', 'T_REQUIRE',
                 'T_REQUIRE_ONCE', 'T_RETURN', 'T_STATIC', 'T_SWITCH', 'T_THROW', 'T_TRAIT', 'T_TRY',
-                'T_UNSET', 'T_USE', 'T_VAR', 'T_WHILE', 'T_YIELD', 'CT_ARRAY_TYPEHINT',
+                'T_UNSET', 'T_USE', 'T_VAR', 'T_WHILE', 'T_YIELD', 'CT_ARRAY_TYPEHINT', 'CT_CLASS_CONSTANT',
```
Comment (GrahamCampbell) [inline Symfony/CS/Tokenizer/Token.php]: Note that this is already present on 2.x.

**Comment by keradus** [comment]
Thank you @tgabi333.

**Comment by SpacePossum** [comment]
thanks @tgabi333 !

**Pull request closed without merging**